src/views/Home/components/Footer/index.view.tsx

```diff
@@ -1,49 +1,38 @@
 import * as React from "react";
+import { ReactComponent as Arrow } from "images/arrow.svg";
+import { footerProps } from "views/Home/props/footer";
```

Comment:
In the name of consistency, let's import these props in `Home` view and pass them in as props to this component.